hpe_3par_kubernetes_manager.py

```diff
@@ -819,9 +843,9 @@ def verify_volume_properties_primera(hpe3par_volume, **kwargs):
             if 'size' in kwargs:
                 if hpe3par_volume['sizeMiB'] != int(kwargs['size']) * 1024:
                     return False
-            else:
-                if hpe3par_volume['sizeMiB'] != 102400:
-                    return False
+                else:

```

Comment:
Changes in indentation makes else of inner if condition, whereas before changes it was for outer if. Request to confirm if this change is intensional.

Reply:
Yes, this was intentional as the subsequent volume property checks were driven based on the "provisioning" option being available in kwargs. This did not work for snapshot volume as the provisioning type was not relevant. So to make it common across base and snapshot volume, made the change to check each individual property if present in kwargs

Reply:
Thanks for confirmation. Approving the PR.